File: boot-mybatis-plus/src/main/java/com/dxp/mybatisplus/Application.java
package com.dxp.mybatisplus;

import org.mybatis.spring.annotation.MapperScan;
import org.mybatis.spring.annotation.MapperScans;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

/**
 * @author carzy.
 * @date 15:04 2019/2/11
 */
@MapperScans({
        @MapperScan("com.dxp.mybatisplus.mapper")
})
@SpringBootApplication
public class Application {

    public static void main(String[] args) {
        SpringApplication.run(Application.class, args);
    }

}

File: boot-shiro/src/main/java/com/dxp/shiro/Application.kt
package com.dxp.shiro

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication

/**
 * 启动类
 *
 * @author carzy
 * @date 2020/8/18
 */
@SpringBootApplication
open class Application

fun main(args: Array<String>) {

    SpringApplication.run(Application::class.java, *args)

}